text = """
￼
LAST, First, M.I. ####-###-### 
PATIENT INFORMATION 
SPECIMEN INFORMATION 
PROVIDER INFORMATION 
￼ ￼
LAST, First, M.I. ID#: DOB: Month, Day, Year Sex: 
Type: Collected: Month, Day, Year Received: Month, Day, Year PG ID: 
Physician Genetic Counselor Institution 
MOLECULAR GENETICS REPORT: Malignant Hyperthermia Susceptibility Panel 
NEGATIVE 
RESULTS AND INTERPRETATIONS: In this patient, for the RYR1, CACNA1S and STAC3 genes, we found no sequence variants that are likely to cause Malignant Hyperthermia (MH) or other Mendelian disorders related to these genes. 
This patient is also apparently negative for copy number variants (CNVs) within the genomic regions of this test. 
These results should be interpreted in context of clinical findings, family history and other laboratory data. All genetic tests have limitations. See limitations and other information for this test on the following pages. 
NOTES: These results reduce, but do not eliminate, the chance that this patient is susceptible to MH. The causes of MH are incompletely understood. 
GENES ANALYZED: CACNA1S, RYR1, STAC3 SUMMARY STATISTICS: 
￼ ￼
Since this test is performed using exome capture probes, a reflex to any of our exome-based tests is available 
￼
(PGxome, PGxome Custom Panels). 
￼ ￼ ￼ ￼
Pipeline Version Average NGS Coverage 
Infinity_Pipeline 1.2.8 273x 
Fraction Bases Covered with NGS 98.8% 
￼ ￼ ￼ ￼ ￼ ￼ ￼ ￼
Minimum NGS coverage is ≥20x for all exons and +/-10bp of flanking DNA, and ≥10x from 11-20bp of flanking DNA. 
Electronically signed on Month, Day, Year by: Electronically signed and reported on Month, Day, Year by: 
￼
ENHANCED GENE LIST (Transcript Number - 100% coverage): CACNA1S (NM_000069), RYR1 
￼
(NM_000540), STAC3 (NM_145064) 
￼ ￼ ￼
1 OF 5 
￼
LAST, First, M.I. ####-###-### 
￼
SUPPLEMENTAL INFORMATION V.18.09 NEXTGEN SEQUENCING 
Limitations and Other Test Notes 
Interpretation of the test results is limited by the information that is currently available. Better interpretation should be possible in the future as more data and knowledge about human genetics and this specific disorder are accumulated. 
When Next Gen or Sanger sequencing does not reveal any difference from the reference sequence, or when a sequence variant is homozygous, we cannot be certain that we were able to detect both patient alleles. Occasionally, a patient may carry an allele which does not capture or amplify, due for example to a large deletion or insertion. 
Where applicable (see below), Copy Number Variants (CNVs) of four exons or more in size are detected through analysis of Next Gen sequence data with sensitivity approaching 100%. Sensitivity however varies from gene to gene based on exon size, depth of coverage, and characteristics of the region. Sensitivity for detection of CNVs smaller than four exons is lower (we estimate ~80%). 
We sequence coding exons for all available transcripts plus 10 bp of flanking non-coding DNA for each exon. We also sequence other regions within or near genes in which pathogenic variants have been identified at PreventionGenetics or reported elsewhere. Unless specifically indicated, test reports contain no information about other portions of genes. 
In most cases, we are unable to determine the phase of sequence variants. In particular, when we find two likely causative mutations for recessive disorders, we cannot be certain that the mutations are on different chromosomes. 
The ability to detect low-level mosaicism of variants is limited. 
Runs of mononucleotide repeats (eg (A)n or (T)n) with n >8 in the reference sequence are generally not analyzed because of strand slippage during amplification. 
Unless otherwise indicated, DNA sequence data is obtained from a specific cell-type (often leukocytes from whole blood). Test reports contain no information about the DNA sequence in other cell-types. 
We cannot be certain that the reference sequences are correct. Genome build hg19, GRCh37 (Feb2009) is used as our reference in nearly all cases. 
We have confidence in our ability to track a specimen once it has been received by PreventionGenetics. However, we take no responsibility for any specimen labeling errors that occur before the sample arrives at PreventionGenetics. 
Genetic counseling to help to explain test results to the patients and to discuss reproductive options is recommended. 
Test Methods 
We use a combination of Next Generation Sequencing (NGS) and Sanger sequencing technologies to cover the full coding regions of the listed genes plus ~10 bases of non-coding DNA flanking each exon. As required, genomic DNA is extracted from the patient specimen. For NGS, patient DNA corresponding to these regions is captured using an optimized set of DNA hybridization probes. Captured DNA is sequenced using Illumina’s 
￼ ￼
2 OF 5 
￼
LAST, First, M.I. ####-###-### 
Reversible Dye Terminator (RDT) platform (Illumina, San Diego, CA, USA). Regions with insufficient coverage by NGS are often covered by Sanger sequencing. 
For Sanger sequencing, Polymerase Chain Reaction (PCR) is used to amplify targeted regions. After purification of the PCR products, cycle sequencing is carried out using the ABI Big Dye Terminator v.3.1 kit. PCR products are resolved by electrophoresis on an ABI 3730xl capillary sequencer. In nearly all cases, cycle sequencing is performed separately in both the forward and reverse directions. 
Copy number variants (CNVs) are detected from NGS data for tests which utilize the exome backbone. Many but not all of our NGS panels use Exome-based probes; please see the test description on the PreventionGenetics website for details. We utilize a CNV calling algorithm that compares mean read depth and distribution for each target in the test sample against multiple matched controls. Neighboring target read depth and distribution and zygosity of any variants within each target region are used to reinforce CNV calls. All reported CNVs are confirmed using another technology such as aCGH, MLPA, or PCR. On occasion, it will not be technically possible to confirm a smaller CNV called by NGS. In these instances, the CNV will not be included on the 
report. 
Human Genome Variation Society (HGVS) recommendations are used to describe sequence variants (http://www.hgvs.org). All differences from the reference sequences are assigned to one of five interpretation categories (Pathogenic, Likely Pathogenic, Variant of Uncertain Significance, Likely Benign and Benign) per ACMG Guidelines (Richards et al. 2015. PubMed ID: 25741868). Rare variants and undocumented variants are nearly always classified as Likely Benign if there is no indication that they alter protein sequence or disrupt splicing. Depending on panel size, Benign or both Benign and Likely Benign variants are not listed in the reports, but are available upon request. 
Data Transfer 
PreventionGenetics recommends that DNA sequence information from this test be stored in the patient’s electronic medical record. This will facilitate reinterpretation of the sequence in future, and will best benefit the patient and family members. Upon request, we will be pleased to transfer the sequence data. 
FDA Notes 
These results should be used in the context of available clinical findings, and should not be used as the sole basis for treatment. This test was developed and its performance characteristics determined by PreventionGenetics. US Food and Drug Administration (FDA) does not require this test to go through premarket FDA review. This test is used for clinical purposes. It should not be regarded as investigational or for research. This laboratory is certified under the Clinical Laboratory Improvement Amendments of 1988 (CLIA) as qualified to perform high complexity clinical laboratory testing. 
￼ ￼
3 OF 5 
￼
LAST, First, M.I. ####-###-### 
￼
SUPPLEMENTAL INFORMATION V.18.06 MALIGNANT HYPERTHERMIA SUSCEPTIBILITY PANEL 
Clinical Features: Malignant Hyperthermia (MH) is a severe adverse reaction to commonly used anesthetics (halothane, sevoflurane, desflurane, enflurane, isoflurane) or to depolarizing muscle relaxants (succinylcholine) (Nelson and Flewellen 1983; Larach et al. 2010; Rosenberg et al. 2013). In susceptible patients these agents may trigger uncontrolled muscle hypermetabolism. In almost all cases, the first manifestations of MH occur in the operating room. Death can result unless the patient is promptly treated. Alternative anesthetics are available for known MH Susceptible individuals. 
Genetics: Malignant Hyperthermia (MH) Susceptibility may be caused by mutations in at least three genes: RYR1, CACNA1S and STAC3. Causative mutations in RYR1 are much more common than causative mutations in the other two genes. 
RYR1 mutations causative for MH are inherited in an autosomal dominant manner, although patients with two causative mutations in trans have been reported (Monnier et al. 2002). All RYR1 MH causative mutations reported in the literature have been either missense or (rarely) in-frame deletion or insertion of one or a few amino acids (Robinson et al. 2006; Ibarra et al. 2006; Levano et al. 2009; www.emhg.org). Premature protein termination, splicing, or large deletion mutations have not been reported in RYR1 for MH (although they are well known in cases of recessive myopathy). Over 300 RYR1 missense variants have been found in MH patients, but only 35-50 have been conclusively demonstrated to be involved in the disease. Most causative RYR1 MH mutations are clustered in three hot spots along the gene. Penetrance of many (and perhaps all) RYR1 MH causative variants is incomplete; that is, MH susceptible patients will not always trigger upon exposure to offending agents (Robinson et al. 2009; Grievink and Stowell 2010). The large RYR1 gene with 106 exons encodes the primary skeletal muscle calcium channel within the sarcoplasmic reticulum membrane. Mutations in RYR1 can also cause a variety of myopathies (see Test #1771). 
Autosomal dominant MH may also be caused by mutations in the CACNA1S gene (Monnier et al. 1997; Carpenter et al. 2009; Toppin et al. 2010). Several CACNA1S missense variants have been connected to MH, but to date the only variant conclusively demonstrated to be involved is c.3257G>A (p.Arg1086His). CACNA1S encodes the α1 subunit of the L-type calcium channel in skeletal muscle, also known as the dihydropyridine receptor. The CACNA1S and RYR1 gene products interact to transduce action potentials into contraction of skeletal muscle fibers (Lanner et al. 2010). Missense mutations in the CACNA1S gene can also cause hypokalemic periodic paralysis. 
Recently, a founder missense mutation in the STAC3 gene has been reported to cause autosomal recessive Native American myopathy (Horstick et al. 2013). This myopathy is characterized by congenital weakness and arthrogryposis, cleft palate, ptosis, short stature, kyphoscoliosis, talipes deformities and susceptibility to MH (Stamm et al. 2008). STAC3 encodes a T-tubule protein which mediates Ca++ release through the RYR1 channel (Nelson et al. 2013). The involvement of STAC3 in MH is not as well established as the other two genes, but we included this gene in our NGS panel to be inclusive. 
Testing Strategy: For this Next Generation Sequencing (NGS) test, sequencing is accomplished by capturing specific regions with an optimized solution-based hybridization kit, followed by massively parallel sequencing of the captured DNA fragments. Additional Sanger sequencing is performed for regions not captured or with insufficient number of sequence reads. 
For Sanger sequencing, polymerase chain reaction (PCR) is used to amplify targeted regions. After purification of the PCR products, cycle sequencing is carried out using the ABI Big Dye Terminator v.3.0 kit. PCR products 
￼ ￼
4 OF 5 
￼
LAST, First, M.I. ####-###-### 
are resolved by electrophoresis on an ABI 3730xl capillary sequencer. In nearly all cases, cycle sequencing is performed separately in both the forward and reverse directions. 
Copy number variants (CNVs) are also detected from NGS data. We utilize a CNV calling algorithm that compares mean read depth and distribution for each target in the test sample against multiple matched controls. Neighboring target read depth and distribution and zygosity of any variants within each target region are used to reinforce CNV calls. All CNVs are confirmed using another technology such as aCGH, MLPA, or PCR before they are reported. 
This panel provides 100% coverage of all coding exons of the genes listed, plus ~10 bases of flanking noncoding DNA. We define coverage as ≥20X NGS reads or Sanger sequencing. 
Since this test is performed using exome capture probes, a reflex to any of our exome based tests is available (PGxome, PGxome Custom Panels). 
Indications for Test: Ideal MH test candidates have a family history of MH along with either a positive in vitro contracture test or a clear MH event. Testing should begin in such a family member. If a causative mutation is identified, other family members can be screened at much reduced cost. Other, less ideal candidates for testing are those with just a family history of MH or those with a “MH-like” event and no family history. 
Sensitivity of Test: Sensitivity for ideal test candidates (family history of MH plus either positive in vitro contracture test or an MH event) is approximately 60% (Monnier et al. 2005; Levano et al. 2009). 
No large deletions or insertions in any of the three genes have been reported in MH patients. 
References: Please see the test description for this test on our website (www.preventiongenetics.com) for a full list of complete citations. 
￼
5 OF 5 

"""
